app/controllers/documents_controller.rb

```diff
@@ -220,10 +220,12 @@ def update
       end
       if params[:publish] == "true" && @organization.enable_workflows && user
         if @document.workflow_step_id && @document.assigned_to?(user)
-          WorkflowMailer.step_email(user, @organization, @document.workflow_step.slug, component_allowed_liquid_variables(@document.workflow_step, user,@organization)).deliver_later
+          WorkflowMailer.step_email(@document, request.env["SERVER_NAME"],user, @organization, @document.workflow_step.slug, component_allowed_liquid_variables(@document.workflow_step, user,@organization)).deliver_later
```

Comment:
This should be the org path (from slugs)

Reply:
ok